test/src/org/labkey/test/tests/targetedms/TargetedMSQCConfigureMetricTest.java

```diff
@@ -73,6 +77,114 @@ public void testInconsistentMetricDisplay()
+
+        configureQCMetrics.setFixedDeviationFromMean(metricType, "-2", "");
+        Assert.assertEquals("Incorrect error for blank upper bound",
+                "Error: For Mean Deviation Cut-Off configuration, you must provide an upper bound",
+                configureQCMetrics.clickSaveExpectingError());
+
+        configureQCMetrics.setFixedDeviationFromMean(metricType, "2", "2");
+        Assert.assertEquals("Incorrect error for non negative lower bound",
+                "Error: For Mean Deviation Cut-Off configuration, the lower bound must be less than 0",
+                configureQCMetrics.clickSaveExpectingError());
+
+        configureQCMetrics.setFixedDeviationFromMean(metricType, "-2", "0");
+        Assert.assertEquals("Incorrect error for negative upper bound",
+                "Error: For Mean Deviation Cut-Off configuration, the upper bound must be greater than 0",
+                configureQCMetrics.clickSaveExpectingError());
+
+        configureQCMetrics.setFixedDeviationFromMean(metricType, "-2", "2");
+        configureQCMetrics.clickSave();
+
+        QCPlotsWebPart qcPlotsWebPart = new PanoramaDashboard(this).getQcPlotsWebPart();
+        qcPlotsWebPart.setMetricType(metricType);
+        qcPlotsWebPart.setScale(QCPlotsWebPart.Scale.LINEAR);
+
+        String replicate = "Q_Exactive_08_09_2013_JGB_87";
+
+        //TODO: add the verification steps.
+    }
+
+    @Test
+    public void testFixedValueCutOffOption()
+    {
+        QCPlotsWebPart.MetricType metric = QCPlotsWebPart.MetricType.TRANSITION_MASS_ERROR;
+        goToProjectHome();
+        new PanoramaDashboard(this).getQcSummaryWebPart().clickMenuItem("Configure QC Metrics");
+        ConfigureMetricsUIPage configureQCMetrics = new ConfigureMetricsUIPage(this);
+
+        configureQCMetrics.setFixedValueCutOff(metric, "5", "-5");
+        Assert.assertEquals("Incorrect error for upper bound < lower bound",
+                "Error: Upper bound must be greater than lower bound",
+                configureQCMetrics.clickSaveExpectingError());
+
+        configureQCMetrics.setFixedValueCutOff(metric, "a", null);
+        Assert.assertEquals("Incorrect error for non integer lower bound",
+                "Error: Unable to convert value 'a' to Number (Double)",
+                configureQCMetrics.clickSaveExpectingError());
+
+        configureQCMetrics.setFixedValueCutOff(metric, "-5", "!@#$%^&*()");
+        Assert.assertEquals("Incorrect error for non integer upper bound",
+                "Error: Unable to convert value '!@#$%^&*()' to Number (Double)",
+                configureQCMetrics.clickSaveExpectingError());
+
+        configureQCMetrics.setFixedValueCutOff(metric, "-5", "5");
+        configureQCMetrics.clickSave();
+        QCPlotsWebPart qcPlotsWebPart = new PanoramaDashboard(this).getQcPlotsWebPart();
+        qcPlotsWebPart.setMetricType(metric);
+        qcPlotsWebPart.setScale(QCPlotsWebPart.Scale.LINEAR);
+        qcPlotsWebPart.waitForPlots(6);
```

Comment:
I think if it is the SCProCP... file, there are usually 7 plots.

Reply:
There are only 6 plots displayed.